src/app/modules/submission/encrypt-submission/encrypt-submission.middleware.ts

```diff
@@ -67,3 +67,45 @@ export const validateEncryptSubmissionParams = celebrate({
     isPreview: Joi.boolean(),
   }),
 })
+
+/**
+ * Celebrate middleware for verifying shape of encrypted submission
+ */
+export const validateStorageSubmissionParams = celebrate({
```

Comment:
this is exactly the same as `validateResponseParams` in `email-submission.middleware`, can consider sharing across the two?

Reply:
it's not exactly the same, this one requires version too!

Reply:
point taken though, let me see if I can refactor

Reply:
Done in [d328c73](https://github.com/opengovsg/FormSG/pull/6587/commits/d328c73a5411168d227190fd7204c17392387b2f)

Reply:
thanks! haha yes i missed out the version bit 😅

Reply:
for posterity, I did it wrong in [d328c73](https://github.com/opengovsg/FormSG/pull/6587/commits/d328c73a5411168d227190fd7204c17392387b2f) by replacing the params in the wrong storage-mode middleware 😓 fixed in [6c85f4f](https://github.com/opengovsg/FormSG/pull/6587/commits/6c85f4f4a7f41dad0876aed8105d52294a45ae3d)!